tools/c2chapel/c2chapel.py

```diff
@@ -61,8 +61,8 @@
 ## be registered here.
 c2chapel = {}
 
-c2chapel["double"] = "c_double"
-c2chapel["float"]  = "c_float"
+c2chapel["double"] = "real(64)"
+c2chapel["float"]  = "real(32)"
```

Comment:
What's the motivation for this change? Seems like we'd want to use the c_ type.

Reply:
The `c_double` was causing an error saying it didn't exist and I found this on the [docs](https://chapel-lang.org/docs/1.16/technotes/extern.html):
```
For consistency, the following type aliases are also provided even though their sizes can't vary in C 
(thereby permitting the equivalent Chapel types to always be usable):

c_float  // (a real(32) in Chapel)
c_double // (a real(64) in Chapel)
```
so I changed those to just be their equivalents and didn't have any problems, so I thought it should be kept. Do you think I should move it back? Would just need to figure out why that c_double and c_float weren't working.

Reply:
I guess, for more background, I figured out they all need SysCTypes when I tried compiling the generated files and none of them worked without that and then the other thing that was preventing them from compiling was the c_double and c_float, so when I put this change in it resolved it and they all compile now. I'm not sure if I'm missing an include or something though

Reply:
Right `c_double` should be an alias for `real(64)`, but since we're using c types it makes sense to me to use the `c_double` alias.

I'm not sure I follow your last 2 comments completely though. What didn't compile before?

Reply:
Trying to run this program:
```chapel
use SysCTypes;

var a:c_double;
```

I get this error:
`tools/c2chapel/test/asd.chpl:3: error: 'c_double' undeclared (first use this function)`

Reply:
I see -- it looks like `c_double` is defined in SysBasic.chpl. I'm not sure why, but if you import that it should work.

Reply:
From the header it says:
> This module provides basic types when those types are not
> platform-dependent. The module :mod:`SysCTypes` contains the
> platform-dependent declarations for C compatibility.

And since like your comment above says c_double isn't system dependent we must stuff it here instead of SysCTypes.

Reply:
Alright, added

Reply:
Can you update PR title/description to reflect this change?

Reply:
Yup, done